test/integration/data/tiplink-text/plugin.js

```diff
@@ -0,0 +1,109 @@
+let redact;
```

Comment:
I copied this directly from `test/integration/data/tiplink/plugin.js`. I don't know if there is a better way to include this or if we're okay with just having a copy of this plugin.

Reply:
this is fine for now; eventually it would be nice to integrate this with https://github.com/code-dot-org/remark-plugins